src/test/resources/com/puppycrawl/tools/checkstyle/checks/imports/unusedimports/InputUnusedImportsFromStaticMethodRef.java

```diff
@@ -0,0 +1,91 @@
+/*
+UnusedImports
+processJavadoc = (default)true
+
+
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.imports.unusedimports;
+
+import static java.lang.annotation.ElementType.TYPE_USE;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+import java.lang.ref.WeakReference;
+import java.util.ArrayList;
+import java.util.Optional;
+import java.util.Objects;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.Collections;
+import java.util.function.Consumer;
+import java.util.function.Function;
+import java.util.function.Supplier;
+import static java.lang.String.format; // violation 'Unused import - java.lang.String.format.'
+import static java.util.Arrays.sort; // violation 'Unused import - java.util.Arrays.sort.'
```

Comment:
can we remove this import ?
Maybe good prove will be just another input file, with same code and wihtout extra imports.
User action on  "Unused import" is just delete a line, is that is not correct in this context, message should be different.

Reply:
ok, I see `import java.util.Arrays;` above. it is ok.